examples/clone-database.jl

```diff
@@ -22,25 +22,21 @@ include("parseargs.jl")
 # Answers if the given value represents a terminal state.
 is_term_state(state) = state == "CREATED" || occursin("FAILED", state)
 
-function run(database, engine, source; profile)
+function run(database, source; profile)
     conf = load_config(; profile = profile)
     ctx = Context(conf)
-    rsp = create_database(ctx, database, engine; source = source)
-    while !is_term_state(get(rsp, "state", ""))  # wait for terminal state
-        sleep(3)
-        rsp = get_database(ctx, database)
-    end
+    # NOTE: create_database is a synchronous operation, so there is no need to poll.
```

Comment:
Do we need the note? If there are other API methods that are asynchronous, then it definitely makes sense to call this out explicitly. If not, then I think omitting this reduces confusion. Same for `create_database()`

Reply:
k thanks, i was wondering that too. I agree with you! I'll make that change. 👍

Reply:
done!